lib/parallel_tests/cli.rb

```diff
@@ -88,6 +88,18 @@ def report_results(test_results)
       results = @runner.find_results(test_results.map { |result| result[:stdout] }*"")
       puts ""
       puts @runner.summarize_results(results)
+      report_failure_rerun_commmand(test_results)
+    end
+
+    def report_failure_rerun_commmand(test_results)
+      failing_sets = test_results.select{|test_result| test_result[:exit_status] != 0 }
+      return unless failing_sets.size > 0
+
+      puts "\n\nTests have failed for a parallel_test group. Use the following command to run the group again:\n\n"
+      failing_sets.each do |failing_set|
+        puts failing_set[:command]
```

Comment:
The output of `failing_set[:command] looks like this:

```
TEST_ENV_NUMBER=;export TEST_ENV_NUMBER;PARALLEL_TEST_GROUPS=4;export PARALLEL_TEST_GROUPS;bundle exec rspec --color --tty -O .rspec_parallel spec/filtering/base_spec.rb
```

It's not the `parallel_test` command, since that isn't easily reproduced. This is the command that caused a failure in a particular group. I will insert the seed command after the `rspec` which I do think is a good upgrade.